guides/source/asset_pipeline.md

```diff
@@ -509,17 +510,16 @@ might concatenate three CSS files together this way:
 ### Preprocessing
 
 The file extensions used on an asset determine what preprocessing is applied.
-When a controller or a scaffold is generated with the default Rails gemset, a
-CoffeeScript file and a SCSS file are generated in place of a regular JavaScript
-and CSS file. The example used before was a controller called "projects", which
-generated an `app/assets/stylesheets/projects.scss` file.
-
-In development mode, or if the asset pipeline is disabled, when these files are
-requested they are processed by the processors provided by the `coffee-script`
-and `sass` gems and then sent back to the browser as JavaScript and CSS
-respectively. When asset pipelining is enabled, these files are preprocessed and
-placed in the `public/assets` directory for serving by either the Rails app or
-web server.
+When a controller or a scaffold is generated with the default Rails gemset, an
+SCSS file is generated in place of a regular CSS file. The example used before
+was a controller called "projects", which generated an
+`app/assets/stylesheets/projects.scss` file.
+
+In development mode, or if the asset pipeline is disabled, when this file is
+requested it is processed by the processor provided by the `sass` gem and then
```

Comment:
The `sass` gem is no longer maintained, and `sass-rails` now depends on the `sassc` gem instead via `sassc-rails` (https://github.com/rails/rails/issues/32896).

Even if it's not technically accurate, I think this should say "provided by the `sass-rails` gem" - for the purposes of following this guide, it's enough to know that the processor is available because `sass-rails` is in the Gemfile, even if the code is ultimately provided by a different gem.

Reply:
👍 Updated to `sass-rails` in 649d265e16b8333f5a86aa3de46ae4aac6635e30